Core/Utilities/MySQL.py

```diff
@@ -1488,7 +1477,7 @@ def insertFields( self, tableName, inFields = None, inValues = None, conn = None
                          ( table, inFieldString, inValueString ), conn, debug = True )
 
 
-  def executeStoredProcedure( self, packageName, parameters, outputIds, output = True, array = None, conn = False ):
+  def executeStoredProcedure( self, packageName, parameters, outputIds ):
```

Comment:
There is, as of today, no connections between the MySQL class and the OracleDB class, so I think this is not an issue. 
We can, on a side note, discuss that this is not ideal, but it won't affect this change.